recirq/quantum_chess/circuit_transformer.py

```diff
@@ -49,7 +49,8 @@ def __init__(self, device: cirq.Device):
         self.qubit_list = device.qubits
 
     def qubits_within(self, depth: int, qubit: cirq.GridQubit,
-                      qubit_list: Iterable[cirq.GridQubit]) -> int:
+                      qubit_list: Iterable[cirq.GridQubit],
+                      visited: Set[cirq.GridQubit]) -> int:
```

Comment:
Can you add the new argument to the docstring below?

Reply:
Yes that's been added.

Reply:
Thanks it's added now.